LPFilter/Source/PluginEditor.cpp

```diff
@@ -1,43 +1,54 @@
-/*
-  ==============================================================================
-
-    This file was auto-generated!
-
-    It contains the basic framework code for a JUCE plugin editor.
-
-  ==============================================================================
-*/
-
-#include "PluginProcessor.h"
-#include "PluginEditor.h"
-
-
-//==============================================================================
-LpfilterAudioProcessorEditor::LpfilterAudioProcessorEditor (LpfilterAudioProcessor& p)
-    : AudioProcessorEditor (&p), processor (p)
-{
-    // Make sure that before the constructor has finished, you've set the
-    // editor's size to whatever you need it to be.
-    setSize (400, 300);
-}
-
-LpfilterAudioProcessorEditor::~LpfilterAudioProcessorEditor()
-{
-}
-
-//==============================================================================
-void LpfilterAudioProcessorEditor::paint (Graphics& g)
-{
-    // (Our component is opaque, so we must completely fill the background with a solid colour)
-    g.fillAll (getLookAndFeel().findColour (ResizableWindow::backgroundColourId));
-
-    g.setColour (Colours::white);
-    g.setFont (15.0f);
-    g.drawFittedText ("Hello World!", getLocalBounds(), Justification::centred, 1);
-}
-
-void LpfilterAudioProcessorEditor::resized()
-{
-    // This is generally where you'll want to lay out the positions of any
-    // subcomponents in your editor..
-}
+/*
+  ==============================================================================
+
+    This file was auto-generated!
+
+    It contains the basic framework code for a JUCE plugin editor.
+
+  ==============================================================================
+*/
+
+#include "PluginProcessor.h"
+#include "PluginEditor.h"
+
+
+//==============================================================================
+LpfilterAudioProcessorEditor::LpfilterAudioProcessorEditor (LpfilterAudioProcessor& p)
+    : AudioProcessorEditor (&p), processor (p)
+{
+    // Add components
+    addAndMakeVisible (bypassButton);
+    bypassButton.addListener(this);
+    bypassButton.setButtonText(processor.bypassParam->name);
+    
+    
+    // Make sure that before the constructor has finished, you've set the
+    // editor's size to whatever you need it to be.
+    setSize (400, 300);
+}
+
+LpfilterAudioProcessorEditor::~LpfilterAudioProcessorEditor()
+{
+}
+
+//==============================================================================
+void LpfilterAudioProcessorEditor::buttonClicked(Button* button)
+{
+    processor.bypassParam->operator=(button->getToggleState()) ;
```

Comment:
- [ ] You might right `*processor.bypassParam = button->getToggleState();`
- [ ] If you add more buttons then you should check which button called `buttonClicked` and update the corresponding parameter